### PR TITLE
Added link to search from homepage (#66)

### PR DESCRIPTION
**Pull request by zarkonite**
ok technically it's an iframe...

### CODE REVIEW

**Comment by jansensan** [comment]
@zarkonite Again, please remember to assign the PR to me, I cannot proceed with the review otherwise.

File: src/scripts/components/download-cta/download-cta-template.html [hunk 1]
```diff
@@ -27,3 +27,13 @@
         </svg>
       </a>
     </div>
+
+    <div class="iframe-wrapper">
+      <div class="search-instructions">
+        <span>
```
Comment (jansensan) [inline src/scripts/components/download-cta/download-cta-template.html]: The `<span>` is a superfluous DOM element, please remove.

File: src/scripts/pages/home/banner/home-banner.less [hunk 2]
```diff
@@ -52,27 +52,44 @@
...
   }
   
-  .search-prompt {
-    position:absolute;
-    z-index:100;
-    width:100%;
-    height:200px;
-    top:150px;
-  }
+  .iframe-wrapper {
```
Comment (jansensan) [inline src/scripts/pages/home/banner/home-banner.less]: Provide a more specific name, this is too generic. `.search-iframe` would be more appropriate.

File: src/scripts/pages/home/banner/home-banner.less [hunk 3]
```diff
@@ -52,27 +52,44 @@
...
+    margin: 20px auto;
+    width: 720px;
+    height: 100px;
+    padding: 30px;
+    position: relative;
+    top: 100px;
+    z-index: 101;
```
Comment (jansensan) [inline src/scripts/pages/home/banner/home-banner.less]: Don't do this, it's a cheap trick that leads to messy code. Figure out what is the necessary `z-index` and assign the proper value.

File: src/scripts/pages/home/banner/home-banner.less [hunk 4]
```diff
@@ -52,27 +52,44 @@
...
+      .search-instructions {
+        padding: 0 0 15px 0;
+        text-align: center;
+        text-transform: uppercase;
+      }
+
+      iframe {
+        border: 0px;
+        width: 420px;
+        z-index: 111;
```
Comment (jansensan) [inline src/scripts/pages/home/banner/home-banner.less]: Same here, don't assign an arbitrarily high value. Also, the iframe being a DOM element in your wrapper, it shoudn't need an additional `z-index`, as the wrapper already was assigned one.

File: src/scripts/pages/home/banner/home-banner.less [hunk 5]
```diff
@@ -52,27 +52,44 @@
...
+        display: block;
```
Comment (jansensan) [inline src/scripts/pages/home/banner/home-banner.less]: `display: block` is the default value of DOM elements, is this necessary or could it work without it? I believe it is a superfluous assignment.

**Comment by zarkonite** [comment]
ah crap! almost perfect...

File: src/scripts/components/download-cta/download-cta-template.html [hunk 6]
```diff
@@ -27,3 +27,13 @@
...
+          Vous pouvez &eacute;galement utiliser notre engin de recherche en ligne
+        </span>
+      </div>
+      <iframe src="https://plan.megapode.ca/standalone-search"></iframe>
```
Comment (jansensan) [inline src/scripts/components/download-cta/download-cta-template.html]: You need to add more attributes to your iframe:

    seamless="seamless"
    frameBorder="0"
    scrolling="no"
    role="region"

File: src/scripts/components/footer/footer-template.html [hunk 7]
```diff
@@ -59,11 +59,6 @@
         <div class="list-header"><a class="footer-link" href="/nous-joindre">Nous joindre</a></div>
       </div>
     </div>
-
-    <div class="download-cta-container">
```
Comment (jansensan) [inline src/scripts/components/footer/footer-template.html]: Why is this removed with this change? It has nothing to do with the ticket, it should be a separate issue. Please change only one feature at a time in a PR, otherwise reviewing the changes becomes too complex.

File: src/scripts/components/download-cta/download-cta-template.html (hunk 1 above)
Comment (zarkonite) [inline src/scripts/components/download-cta/download-cta-template.html]: Copied the structure that was made for the menu, don't know either why you'd use a span but I assumed it was best practice since it came from you.

File: src/scripts/pages/home/banner/home-banner.less (hunk 4 above)
Comment (zarkonite) [inline src/scripts/pages/home/banner/home-banner.less]: It needs it because it expands past the container box depending on the # of results, should stay otherwise won't work.

File: src/scripts/components/footer/footer-template.html (hunk 7 above)
Comment (zarkonite) [inline src/scripts/components/footer/footer-template.html]: The element is loaded in both the footer and the banner. Don't have time to fix this so removed it, priority is getting the search out.

File: src/scripts/pages/home/banner/home-banner.less (hunk 5 above)
Comment (zarkonite) [inline src/scripts/pages/home/banner/home-banner.less]: Without it margin:auto doesn't work.
